core/src/main/resources/struts-default.xml

```diff
@@ -376,6 +377,10 @@
                 <interceptor-ref name="alias"/>
                 <interceptor-ref name="servletConfig"/>
                 <interceptor-ref name="i18n"/>
+                <interceptor-ref name="cspInterceptor">
+                    <param name="enforcingMode">true</param>
```

Comment:
We should set this to false by default so it's report-only -but when I installed the core with this version my headers were still set to report-only so we should look into why this might not be working